pkg/controller/user/update_test.go

```diff
@@ -177,7 +177,7 @@ func TestHandleUpdate(t *testing.T) {
 		t.Parallel()
 
 		browserCtx := browser.New(t)
-		taskCtx, done := context.WithTimeout(browserCtx, 30*time.Second)
+		taskCtx, done := context.WithTimeout(browserCtx, 120*time.Second)
```

Comment:
feels like all of these should be a var somewhere so next time we can change it once...

don't have to be in this PR